app/search/query/tests/test_yahoo.py

```diff
@@ -0,0 +1,79 @@
+from unittest import TestCase
+from unittest.mock import patch
+from urllib.parse import parse_qs, urlparse
+
+import responses
+from nose.tools import eq_, ok_
+
+from app.search.query.yahoo import YahooQueryEngine
+from app.tests.memcached import mock_memcached
+
+
+QUERY = 'loveland'
+MOCK_RESPONSE = {
+    'bossresponse': {
+        'web': {
+            'count': '2',
+            'start': '0',
+            'results': [
+                {
+                    'abstract': 'Welcome to <b>Loveland</b> Ski Area Near...',
+                    'url': 'http://skiloveland.com/',
+                    'title': '<b>Loveland</b> Ski Area :: Colorado Ski...',
+                    'date': '',
+                    'dispurl': 'ski<b>loveland</b>.com',
+                    'clickurl': 'http://skiloveland.com/'
+                },
+                {
+                    'abstract': 'The City of <b>Loveland</b> is the Home...',
+                    'url': 'https://en.wikipedia.org/wiki/Loveland,_Colorado',
+                    'title': '<b>Loveland</b>, Colorado - Wikipedia',
+                    'date': '',
+                    'dispurl': 'https://en.wikipedia.org/wiki/<b>Loveland...',
+                    'clickurl': 'https://en.wikipedia.org/wiki/Loveland...'
+                }
+            ],
+            'totalresults': '2'
+        },
+        'responsecode': '200'
+    }
+}
+
+
+class TestYahooClassifier(TestCase):
+    def setUp(self):
+        self.instance = YahooQueryEngine('')
+
+    def test_oauth_url(self):
```

Comment:
I don't think so; that seems like an integration-type test. It'd [raise an exception](https://github.com/chuckharmston/universal-search-recommendation/blob/2-unit-tests/app/main.py#L22-L25) in Flask would catch and return details for in debug mode. Once deployed, I'd expect that to be logged to Sentry for further investigation.